docs/styles.md

```diff
@@ -2,80 +2,507 @@
 
 In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below.
```

Comment:
A couple of comments related to doc updates:

1. Now that this doc is starting to get kind of long, what do you think about adding a ToC like the following to the top of the doc (you may want to use a different structure / labels -- this is just to illustrate the suggestion):

```markdown
Style classes:

* Tables
  * [.os-raise-textheavytable](#text-heavy-table)
  * [.os-raise-horizontaltable](#horizontal-table)
  * [.os-raise-midsizetable](#mid-size-table)
  * `.os-raise-skinnytable`
    * [Example: .os-raise-skinnytable with media](#skinny-table-table--media)
    * [Example: .os-raise-skinnytable multiple tables](#skinny-table-three-tables)
  * [.os-raise-wideequaltable](#wide-equal-column-table)
  * [.os-raise-wideadjustedtable](#wide-adjusted-table)
  * [.os-raise-doubleheadertable](#double-header-table)
* Spacing
  * [.os-raise-indent](#indent-os-raise-indent)
  * [.os-raise-noindent](#no-indent-os-raise-noindent)
  * [.os-raise-text-center](#centered-text-os-raise-text-center)
* Layout
  * [.os-raise-d-flex](#side-by-side-os-raise-d-flex)
  * [.os-raise-align-items-center](#side-by-side-os-raise-d-flex)
  * [.os-raise-justify-content-center](#side-by-side-os-raise-d-flex)
  * [.os-raise-justify-content-between](#side-by-side-os-raise-d-flex)
  * [.os-raise-justify-content-evenly](#side-by-side-os-raise-d-flex)
* Components
  * [.os-raise-graybox](#gray-box-os-raise-graybox)
  * [.os-raise-motivation](#motivational-content-os-raise-motivation)
```

2) Let's add documentation for the two new utility classes you have including `.os-raise-mb-0` and `.os-raise-mx-auto`

Reply:
My thoughts on your points below:

1. I think a ToC makes sense. I will incorporate one into the markdown.

2. I realized I forgot to include those in the documentation after I pushed the commit. Just finished my suggested documentation adds for those two utility classes.